swagger.yml

```diff
@@ -5,6 +5,9 @@ info:
     name: Tyk Technologies
     url: https://tyk.io/contact
   description: |+
+    <img src="https://content.pstmn.io/260470c2-eff2-4fe0-82e2-7e18feefb40e/aW1hZ2UucG5n" width="963" height="250">
```

Comment:
Can we host the image on our Docs repository to maintain control over it? If the image needs to be changed in the future, we can update it on our servers and it will be automatically updated.